lib/MusicBrainz/Server/Report/TracksWithSequenceIssues.pm

```diff
@@ -35,12 +34,11 @@ FROM
       GROUP BY track.medium
    ) s
    JOIN medium ON medium.id = s.medium
-   LEFT JOIN medium_format ON medium_format.id = medium.format
    JOIN release ON release.id = medium.release
    WHERE
-     (first_track != 1 AND NOT (first_track = 0 AND (medium_format.id IS NULL OR medium_format.has_discids))
-     OR (last_track != s.track_count AND NOT (last_track == s.track_count - 1 AND (medium_format.id IS NULL OR medium_format.has_discids))
-     OR (s.track_count * (1 + s.track_count)) / 2 <> track_pos_acc
+     (first_track <> 1 AND first_track <> 0)
+     OR last_track <> s.track_count - (1 - first_track)
+     OR (last_track * (1 + last_track)) <> 2 * track_pos_acc
```

Comment:
Using `last_track` instead of `track_count` because it isn’t influenced by the presence or absence of a pre-gap track. It would be possible to replace `last_track` (twice) with `s.track_count - (1 - first_track)`.